### PR TITLE
Remove dupicate doc

### PR DESCRIPTION
**Pull request by richsalz**
Issue found by build for #3064.  Unrelated to that PR.

### CODE REVIEW

File: doc/man3/SSL_CTX_set_ex_data.pod [hunk 1]
```diff
@@ -19,14 +19,10 @@ SSL_SESSION_get_ex_data, SSL_SESSION_set_ex_data
 
  int SSL_set_ex_data(SSL *s, int idx, void *arg);
 
- int SSL_SESSION_set_ex_data(SSL_SESSION *s, int idx, void *arg);
-
- void *SSL_SESSION_get_ex_data(const SSL_SESSION *s, int idx);
-
```
Comment (levitte) [inline doc/man3/SSL_CTX_set_ex_data.pod]: Remove them from the NAME section as well.

Comment (richsalz) [inline doc/man3/SSL_CTX_set_ex_data.pod]: Doh!  drat.  fixed and updated commit pushed

**Comment by levitte** [comment]
Although, *not* 1.0.2, since #3050 didn't go there

**Comment by levitte** [comment]
Merged.

master: a01dbac232166563cdeafcbe31f321c05d164a10
1.1.0: 1d1255fe992efe2c65db7e0a6722b55ede03034a

**Pull request closed without merging**